src/finding/mod.rs

```diff
@@ -221,6 +221,7 @@ pub(crate) struct Finding<'w> {
     pub(crate) desc: &'static str,
     pub(crate) determinations: Determinations,
     pub(crate) locations: Vec<Location<'w>>,
+    pub(crate) ignored_from_inlined_comment: bool,
```

Comment:
Let's shorten this, since `ignored` is unambiguous in this context:

```suggestion
    pub(crate) ignored: bool,
```

Reply:
Done [here](https://github.com/woodruffw/zizmor/pull/187/commits/0de3177ed5dd2f3e944b3e2edc527742e4531a5e)